development/circular-deps.ts

```diff
@@ -1,8 +1,18 @@
-#!/usr/bin/env tsx
+#!/usr/bin/env -S node --require "./node_modules/tsx/dist/preflight.cjs" --import "./node_modules/tsx/dist/loader.mjs"
```

Comment:
Not really necessary, but this is how `tsx` is used in some of our other modules (in webpack). Only reason for doing it this way is that it is _slightly_ faster.

We don't actually need it here (or in webpack), since we aren't launching these scripts directly. I guess it's more of a "technically correct" than actually useful.

In otherwords: it is safe to ignore this change, safe to revert it to the way it was, and even safe to remove the line complete.